src/products/mozilla_org/testing.md

```diff
@@ -30,11 +30,11 @@ It’s highly advised you to ask other community members to conduct peer review
 * Header line break wraps at proper place.
 * Text not overlapping graphic.
 * Terminology consistent with product, and among web pages.
-* Brand names remain in English.
+* Brand names remain in English, with a few exceptions due to [declension](https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#declensions). Check out [these detailed guidelines](https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark) when working with brands and trademarks.
 * Product names comply to Mozilla guideline and adhere to what the community has agreed to.
 * No corrupted characters.
-* Click on the links on the page, which should take you to the pages of the same language if they are localized, or they will be redirected to en-US if the pages are not..
-* Nav bar terms consistent with the page titles they are linked to (except when Nav bar term is shortened due to space limitation).
+* Click on the links on the page, which should take you to the pages of the same language if they are localized, or they will be redirected to en-US if the pages are not.
```

Comment:
```suggestion
* Click on the links on the page, which should take you to the pages of the same language if they are localized, or they will be redirected to en-US if the pages are not.
```

This is not completely accurate. en-US will be the ultimate fallback, but there might be other locales depending on the user's settings (e.g. they ask for content in Fulah, French, English, they will get French before English).

Reply:
Reworded this section to incorporate your input.